jbpm-db-scripts/src/main/resources/db/ddl-scripts/oracle/oracle-springboot-jbpm-schema.sql

```diff
@@ -33,7 +33,7 @@ create sequence VAR_INST_LOG_ID_SEQ start with 1 increment by  1;
 create sequence WORKITEMINFO_ID_SEQ start with 1 increment by  50;
 create table Attachment (id number(19,0) not null, accessType number(10,0), attachedAt timestamp, attachmentContentId number(19,0) not null, contentType varchar2(255 char), name varchar2(255 char), attachment_size number(10,0), attachedBy_id varchar2(255 char), TaskData_Attachments_Id number(19,0), primary key (id));
 create table AuditTaskImpl (id number(19,0) not null, activationTime timestamp, actualOwner varchar2(255 char), createdBy varchar2(255 char), createdOn timestamp, deploymentId varchar2(255 char), description varchar2(255 char), dueDate timestamp, lastModificationDate timestamp, name varchar2(255 char), parentId number(19,0) not null, priority number(10,0) not null, processId varchar2(255 char), processInstanceId number(19,0) not null, processSessionId number(19,0) not null, status varchar2(255 char), taskId number(19,0), workItemId number(19,0), end_date timestamp, primary key (id));
```

Comment:
Missing end_date removal

```suggestion
create table AuditTaskImpl (id number(19,0) not null, activationTime timestamp, actualOwner varchar2(255 char), createdBy varchar2(255 char), createdOn timestamp, deploymentId varchar2(255 char), description varchar2(255 char), dueDate timestamp, lastModificationDate timestamp, name varchar2(255 char), parentId number(19,0) not null, priority number(10,0) not null, processId varchar2(255 char), processInstanceId number(19,0) not null, processSessionId number(19,0) not null, status varchar2(255 char), taskId number(19,0), workItemId number(19,0), primary key (id));
```